test/jdk/java/lang/module/exeNullCallerGetResource/NullCallerGetResource.java

```diff
@@ -25,6 +25,7 @@
 /**
  * @test
  * @bug 8281006
+ * @bug 8281001
```

Comment:
```suggestion
 * @bug 8281006 8281001
```

`@bug` can have one or more bug IDs